rhel8/overlays/stig_overlay.xml

```diff
@@ -0,0 +1,1367 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<overlays xmlns="http://checklists.nist.gov/xccdf/1.1">
+  <overlay owner="disastig" ruleid="installed_OS_is_vendor_supported" ownerid="RHEL-08-010000" disa="366" severity="high">
+    <VMSinfo VKey="230221" SVKey="230221r5997" VRelease="r599732"/>
+    <title text="RHEL 8 must be a vendor-supported release."/>
+  </overlay>
+  <overlay owner="disastig" ruleid="XXXX" ownerid="RHEL-08-010010" disa="366" severity="medium">
```

Comment:
I'm not very familiar with the stig overlay, are these `ruleid="XXXX"` expected?

Reply:
Yes, when there is no rule with corresponding with `ownerid` in the references section (`STIGID`), the script `utils/create-stig-overlay.py` puts this value instead.